dotnet-desktop-guide/framework/wpf/controls/how-to-create-apply-template.md

```diff
@@ -35,11 +36,9 @@ On the other hand, if you need a control with new functionality, different prope
 
 Create a new WPF application and in *MainWindow.xaml* (or another window of your choice) set the following properties on the **\<Window>** element:
 
-|     |     |
-| --- | --- |
-| **Title**         | `Template Intro Sample` |
-| **SizeToContent** | `WidthAndHeight` |
-| **MinWidth**      | `250` |
+| Title | SizeToContent | MinWidth |
+|-------|---------------|----------|
+| `Template Intro Sample` | `WidthAndHeight` | `250` |
```

Comment:
This is better the other way around. You can use Property / Value for the headers
```suggestion
| Property          | Value                   |
| ----------------- | ----------------------- |
| **Title**         | `Template Intro Sample` |
| **SizeToContent** | `WidthAndHeight`        |
| **MinWidth**      | `250`                   |
```